graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/tweet/TopSecondDegreeByCountTweetRecsGenerator.java

```diff
@@ -138,27 +138,25 @@ private static boolean isLessThanMinUserSocialProofSize(
     int minUserSocialProofSize) {
     int length = validSocialProofs.length;
     long authorId = getAuthorId(socialProofs);
+    boolean isLessThan = false;
```

Comment:
nit: In this generator class, we've opted for returning true/false directly, instead of using a conditional variable. Probably should keep the style consistent unless there's a good use case here